docs/docs/widgets/timeline.md

```diff
@@ -34,58 +35,57 @@ The **Timeline** component can be used to do a visual representation of a sequen
 
 ### Hide date
 
-Hide date can be used to hide the date time of the Timeline component.
+Toggle this option to hide the date/time information in the Timeline component.
 
 </div>
 
-<div style={{paddingTop:'24px', paddingBottom:'24px'}}>
+<div style={{paddingTop:'24px'}}>
 
-## Component Specific Actions (CSA)
+## Component-Specific Actions (CSA)
 
-There are currently no CSA (Component-Specific Actions) implemented to regulate or control the component.
+There are currently no Component-Specific Actions implemented for the Timeline component.
 
 </div>
 
-<div style={{paddingTop:'24px', paddingBottom:'24px'}}>
+<div style={{paddingTop:'24px'}}>
 
 ## Exposed Variables
 
-There are currently no exposed variables for the component.
+There are currently no exposed variables for the Timeline component.
 
 </div>
 
-<div style={{paddingTop:'24px', paddingBottom:'24px'}}>
+<div style={{paddingTop:'24px'}}>
 
 ## General
 ### Tooltip
 
-A Tooltip is often used to specify extra information when the user hovers the mouse pointer over the component.
-
-Under the <b>General</b> accordion, you can set the value in the string format. Now hovering over the component will display the string as the tooltip.
+A Tooltip provides additional information when users hover over the component. Set the tooltip content under the **General** accordion in the component's properties.
 
 </div>
 
-<div style={{paddingTop:'24px', paddingBottom:'24px'}}>
+<div style={{paddingTop:'24px'}}>
 
-## Layout
+## Devices
 
-| <div style={{ width:"100px"}}> Layout </div> | <div style={{ width:"100px"}}> Description </div> | <div style={{ width:"135px"}}> Expected Value </div> |
-|:--------------- |:----------------------------------------- | :------------------------------------------------------------------------------------------------------------- |
-| Show on desktop | Toggle on or off to display desktop view. | You can programmatically determining the value by clicking on `Fx` to set the value `{{true}}` or `{{false}}` |
-| Show on mobile  | Toggle on or off to display mobile view.  | You can programmatically determining the value by clicking on `Fx` to set the value `{{true}}` or `{{false}}` |
+| Property        | Description                               | Expected Value |
+| :-------------- | :---------------------------------------- | :------------- |
+| Show on desktop | Controls the component's desktop visibility | Set to `{{true}}` to show on desktop, `{{false}}` to hide |
```

Comment:
For the expected value column, please use the same content provided in the [reference](https://docs.tooljet.com/docs/widgets/text-input/#devices). Edit this in both the rows.

Reply:
@Asymtode712 I think this issue is yet to be resolved.